packages/components/link/src/link.ts

```diff
@@ -26,6 +26,14 @@ export const linkProps = buildProps({
    * @description same as native hyperlink's `href`
    */
   href: { type: String, default: '' },
+  /**
+   * @description same as native hyperlink's `target`
+   */
+  target: {
+    type: String,
+    values: ['_blank', '_parent', '_self', '_top'],
+    default: '_self',
+  },
```

Comment:
## Suggestion

There should be no `values` ​​attribute.

This is because the `iframe` name can come as the target value.

## Docs

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target